jwst/pipeline/outlier_detection.cfg

```diff
@@ -13,3 +13,5 @@ snr = '4.0 3.0'
 scale = '0.5 0.4'
 backg = 0.0        
 save_intermediate_results = False
+resample_data = True
+good_bits = 4
```

Comment:
One reason for this value would be to ignore any previously identified CRs so that this algorithm can identify them independently.  However, if someone (like the JWST CWG) has suggestions on what bits should be considered good, we can use that value instead.